### ISSUE TITLE
Release date for tensorboard in pypi

### ISSUE THREAD

**Issue by cdancette** [dmlc/tensorboard#58]
Hi, 
Thanks for the amazing work,

Do you have an expected release date for this package in pypi ? It's hard to recommend this package to other people for now since it has to be built from source, which is not very easy.

**Comment by edmBernard** [comment]
The Pypi repository is no longer this version of tensorboard : https://github.com/dmlc/tensorboard/issues/50

**Comment by cdancette** [comment]
@zihaolucky I don't need another logger, I already use tensorboar dX (https://github.com/lanpa/tensorboard-pytorch). What I want is an alternative to tensorboard, because it has a tensorflow dependancy (and I don't want to make people install tensorflow just for drawing graphs).

Your package is also a logger, thus needs tensorboard and tensorflow.

**Comment by zihaolucky** [comment]
Indeed, it's always our motivation and objective. We used to use Travis to build and publish the PyPI package, then it's super easy to use, but this solution takes too long to satisfy the Travis restriction in building time.

The maintenance of current solution still remains a problem as we have to include the TensorFlow/TensorBoard as submodule.
